src/index.html

```diff
@@ -21,7 +24,396 @@
+        </a>
+      </div>
+      <div class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__image"
+          alt="imac"
+        />
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__code">Product code: 195434</div>
+
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars__star--inactive"></div>
+
+          <span class="reviews">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="price__text">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <a
+          class="button-buy"
+          href="#"
+        >
+          buy
+        </a>
+      </div>
+      <div class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__image"
+          alt="imac"
+        />
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__code">Product code: 195434</div>
+
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars__star--inactive"></div>
+
+          <span class="reviews">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="price__text">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <a
+          class="button-buy"
+          href="#"
+        >
+          buy
+        </a>
+      </div>
+      <div class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__image"
+          alt="imac"
+        />
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__code">Product code: 195434</div>
+
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars__star--inactive"></div>
+
+          <span class="reviews">Reviews: 5</span>
+        </div>
+
+        <div class="price">
+          <span class="price__text">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <a
+          class="button-buy"
+          href="#"
+        >
+          buy
+        </a>
+      </div>
```

Comment:
I noticed that you have repeated the same structure for the product card multiple times. This is not an issue for a small project like this, but for larger projects, it can make your code harder to maintain. If you need to make a change, you have to do it in multiple places. Consider using a template engine like EJS or Handlebars to create reusable components. This way, you only need to define the structure of a product card once, and then you can reuse it as many times as you need.